README.md

```diff
@@ -97,17 +97,26 @@ The server interface defined in `gen-go/server/interface.go` has one method for
 
 
 ### Logging
-  The [kayvee middleware logger](https://godoc.org/gopkg.in/Clever/kayvee-go.v5/middleware) is automatically added to the context object.
+  The [kayvee middleware logger](https://godoc.org/gopkg.in/Clever/kayvee-go.v6/middleware) is automatically added to the context object.
   It can be pulled out of the context object and used via the kayvee `FromContext` method:
 
 ```go
-import "gopkg.in/Clever/kayvee-go.v5/logger"
+import "gopkg.in/Clever/kayvee-go.v6/logger"
 ...
 logger.FromContext(ctx).Info(...)
 ```
 
   You should use this logger for all logging within your controller implementation.
 
+#### Application Log Routing
```

Comment:
Added a note here https://github.com/Clever/wag/pull/151/commits/48b8300ea26659e1b5c48c94db4dd10d5c867722

Also going to add empty kvconfig.yml to template-wag